display.py

```diff
@@ -0,0 +1,53 @@
+from comms import Comms, Message
+from datetime import datetime
+from datetime import time
+import json
+import multiprocessing as mp
```

Comment:
some of these imports are unused. Should cut them out and alphabetize before final pr.